modules/installation-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,160 @@
+====
+
+The concept of defining zones and regions for your cluster relies on the creation of vCenter tags. Each tag is attached to a vCenter object. Each vCenter object associates with a region and zone. 
+
+The following sequence of steps outlines an example of installing an {product-title} cluster in a vSphere environment that consists of multiple datacenters running in a single VMware vCenter. The procedure assumes that you followed the installer-provisioned infrastructure path. 
+
+[IMPORTANT]
+====
+The example uses the `govc` command. The `govc` command is an open-source command available from VMware; it is not available from Red Hat. The Red Hat support team does not maintain the `govc` command. Instructions for downloading and installing `govc` are found on the VMware documentation website
+====
+
+.Example procedure
+
+. Use the `govc` command-line tool to create two vCenter tag categories named `openshift-region` and `openshift-zone`:
++
+[source,terminal]
+----
+$ govc tags.category.create -d "OpenShift region" openshift-region
+
+$ govc tags.category.create -d "OpenShift zone" openshift-zone
+----
+
+. Create the `us-east` and `us-west` region tags:
++
+[source,terminal]
+----
+$ govc tags.create -c openshift-region us-east
+
+$ govc tags.create -c openshift-region us-west
+----
+
+. Create four zone tags:
++
+[source,terminal]
+----
+$ govc tags.create -c openshift-zone us-east-1a
+
+$ govc tags.create -c openshift-zone us-east-2a
+
+$ govc tags.create -c openshift-zone us-east-3a
+
+$ govc tags.create -c openshift-zone us-west-1a
+----
+
+. Attach region tags to vCenter datacenter objects:
++
+[source,terminal]
+----
+$ govc tags.attach -c openshift-region us-east /datacenter-1
+
+$ govc tags.attach -c openshift-region us-west /datacenter-2
+----
+
+. Attach the zone tags to the vCenter datacenter objects:
++
+[source,terminal]
+----
+$ govc tags.attach -c openshift-zone us-east-1a /datacenter-1/host/vcs-mdcnc-workload-1
+
+$ govc tags.attach -c openshift-zone us-east-2a /datacenter-1/host/vcs-mdcnc-workload-2
+
+$ govc tags.attach -c openshift-zone us-east-3a /datacenter-1/host/vcs-mdcnc-workload-3
+
+$ govc tags.attach -c openshift-zone us-west-1a /datacenter-2/host/vcs-mdcnc-workload-4
+----
+
+. Create the `install-config.yaml` file and configure the file to support operating multiple datacenters in a vSphere center. The following example does not show a complete sample `install-config.yaml` file.
++
+[source,yaml]
+----
+...
+compute:
+...
+  vsphere:
+      zones:
+        - <failure_domain1_name>
+        - <failure_domain2_name>
+...
+controlPlane:
+...
+vsphere:
+      zones:
+        - <failure_domain1_name>
+        - <failure_domain2_name>
+...
+  vsphere:
+    vcenters:
+...
+    datacenters:
+      - <datacenter1_name>
+      - <datacenter2_name>
+    failureDomains:
+    - name: <failure_domain1_name>
+      region: <region1_name>
+      zone: 1
```

Comment:
Based on slide 9 [1] I think `zone` should specify a zone tag in the region. What is unclear to me, is if a `failureDomain` is required for each zone in region. For example, if my datacenter has 3 clusters in it, would I need a `failureDomain` for cluster 1,2, and 3.

[1] https://docs.google.com/presentation/d/1RdRioExW6oAbgfK0DWytR-YHXES2DlHZBpm7x8DTnis/edit#slide=id.g17322bb9019_0_17

Reply:
Hi @jcpowermac . Would you be OK to chime in on this? One ` failureDomains:` spec covers all datacenters and their clusters?